WebRoot/WEB-INF/tags/infra/view.tag

```diff
@@ -414,7 +414,7 @@ if (screen.width<=800) {
 	
 	<c:if test="${adsOn}" >
 					<td valign="top" style="border-top:1px solid #98adbe; width:180px;">
-						<app:ads content="${ads}"/>
+						<app:ads content="${fn:escapeXml(ads)}"/>
```

Comment:
as discussed, this will do double escaping

Reply:
This is fixed now. Removed from view.tag and added escapexml in leaf file that is ads.tag